packages/mobile/src/escrow/actions.ts

```diff
@@ -88,3 +105,35 @@ export const resendPayment = (paymentId: string): ResendPaymentAction => ({
   type: Actions.RESEND_PAYMENT,
   paymentId,
 })
+
+export const setReclaimTransactionFee = (suggestedFee: string): SetReclaimTransactionFeeAction => ({
+  type: Actions.SET_RECLAIM_TRANSACTION_FEE,
+  suggestedFee,
+})
+
+export const fetchReclaimSuggestedFee = (paymentID: string) => async (
```

Comment:
New async actions should all use redux saga instead of redux thunk. Please convert this to saga.